### DISCUSSION TITLE
Hacktoberfest 2021?

### DISCUSSION THREAD

**Discussion by arichar6** [PlasmaPy/PlasmaPy#1290] [General]
Any thoughts about PlasmaPy participating in Hacktoberfest 2021? 

https://hacktoberfest.digitalocean.com/

It's a pretty fun way to encourage open-source software contributions.

**Comment by StanczakDominik** [comment]
We're participating as of... now! Might not have too much time for reviews, but there is a `hacktoberfest-accepted` label in that case.
